### PR TITLE
Replace use of deprecated decode encode string

### PR DESCRIPTION
**Pull request by graingert**
### Contributor Checklist:

- [x] I have updated the release notes at `docs/source/NEWS.rst`
- [ ] I have updated the automated tests.
- [ ] All tests pass on your local dev environment. See `CONTRIBUTING.rst`.

### CODE REVIEW

**Review by adiroiban — APPROVED** [review]
Looks good. Thanks.

I don't think that this change is worth being mentioned in the release notes.
We have the git log for low-level changes.

File: docs/source/NEWS.rst
Comment (graingert) [inline docs/source/NEWS.rst:9]: I think it's worth including the changelog - as it makes the code work on Py3.9. But I don't want to advertise py3.9 support without CI support: https://github.com/twisted/ldaptor/pull/210

```suggestion
- replace deprecated calls to ``base64.decodestring`` and ``base64.encodestring``,
  to support Python 3.9. Official support for Python 3.9 pending a new twisted release.
```

**Comment by graingert** [comment]
merging this now, will decide on the changelog at the next release - eg if twisted does ship a 3.9 supporting build before we do

**Comment by adiroiban** [comment]
I guess that ldaptor can have a release with support 3.9 before Twisted.

As long as the tests are green on Python 3.9 it means that ldaptor is ready for 3.9.
I don't see why ldaptor should be blocked waiting for a Twisted release.

So if #210 is green I think it can be merged and if someone wants to run ldaptor on 3.9 we can also have a new release.

File: docs/source/NEWS.rst
Comment (adiroiban) [reply docs/source/NEWS.rst:9]: I would say that this PR could have been merged without a release note.

In the #210 PR the release note can read: 'Ldaptor now supports Python 3.9'

But is ok. Thanks

**Pull request merged**
